doc/source/configuration/block-storage/drivers/lightbits-lightos-driver.rst

```diff
@@ -141,3 +142,18 @@ Lightbits LightOS Cinder driver.
    :config-target: Lightbits LightOS
 
    cinder.volume.drivers.lightos
+
+Active active supports
+~~~~~~~~~~~~~~~~~~~~~~
+In order to enable active active support, the following configuration
```

Comment:
consider putting a link to the overall cinder active active doc, it's more friendly to the user
Also, then you can say something like "In order to enable active active support, please follow the guidelines at <URL>, including enabling a DLM (Distributed Lock Manager) if needed for your environment, and then add the following configuration options to the cinder.conf file:
...
"